packages/protocol/contracts/thirdparty/risczero/Util.sol

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 RISC Zero, Inc.
```

Comment:
Same as above. Not needed to bring it over as a copy, reference with `npm` or `git` `submodule`.

Reply:
Used pnpm package to keep it outside.